sdk/keyvault/keyvault-admin/src/accessControlModels.ts

```diff
@@ -1,27 +1,95 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import * as coreHttp from "@azure/core-http";
+import { CommonClientOptions, OperationOptions } from "@azure/core-client";
 import { SUPPORTED_API_VERSIONS } from "./constants";
 import {
   DataAction as KeyVaultDataAction,
-  KnownDataAction as KnownKeyVaultDataAction,
-  KnownRoleScope as KnownKeyVaultRoleScope,
   RoleScope as KeyVaultRoleScope
 } from "./generated/index";
 
-export { KeyVaultDataAction, KnownKeyVaultDataAction, KeyVaultRoleScope, KnownKeyVaultRoleScope };
+export { KeyVaultDataAction, KeyVaultRoleScope };
 
 /**
  * The optional parameters accepted by the Key Vault's AccessControlClient
  */
-export interface AccessControlClientOptions extends coreHttp.PipelineOptions {
+export interface AccessControlClientOptions extends CommonClientOptions {
   /**
    * The accepted versions of the Key Vault's service API.
    */
   serviceVersion?: SUPPORTED_API_VERSIONS;
 }
 
+/** Known values of {@link DataAction} that the service accepts. */
+export enum KnownKeyVaultDataAction {
```

Comment:
Moved these here while https://github.com/Azure/autorest.typescript/issues/1013 is investigated